docs/src/main/sphinx/connector/hive.rst

```diff
@@ -393,6 +387,48 @@ Property Name                                      Description
                                                    is limited to number of buckets.
 ================================================== ============================================================ ============
 
+ORC format configuration properties
+-----------------------------------
+
+The following properties can be use to configure the read and write operations
+with ORC files performed by the Hive connector.
+
+.. list-table:: ORC format configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - ``hive.orc.time-zone``
+      - Sets the default time zone for legacy ORC files that did not declare a
+        time zone.
+      - JVM default
+    * - ``hive.orc.use-columns-names``
+      - Accessing ORC columns by name. By default, columns in ORC files are
+        accessed by their ordinal position in the Hive table definition. The
+        equivalent catalog session property is ``orc_use_column_names``.
+      - ``false``
+
+Parquet format configuration properties
+---------------------------------------
+
+The following properties can be use to configure the read and write operations
+with ORC files performed by the Hive connector.
+
+.. list-table:: Parquet format configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - ``hive.parquet.time-zone``
+      - Adjusts timestamp values to a specific time zone. For Hive 3.1+, this
```

Comment:
For Hive 3.1+, set this to UTC. 
(Banish "should be" forever)

Reply:
Fixed